src/components/ValueInput/ValueInput.js

```diff
@@ -268,6 +269,14 @@ export const ValueInputComponent = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [errorMessage]);
 
+  React.useEffect(() => {
+    if (onValueChange) {
+      onValueChange('0');
+      setValueInFiat('0');
+    }
```

Comment:
```suggestion
    onValueChange?.('0');
    setValueInFiat('0');
```

Reply:
@mdjastrzebski Thank you for reviewing. The changes has been done.